application-development/helloworld-springboot-auth/src/main/appengine/app.yaml

```diff
@@ -0,0 +1,30 @@
+# [START_EXCLUDE]
+# Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
can remove All Rights Reserved.

Should also remove the  

# [START_EXCLUDE]
+# [END_EXCLUDE]

comments which I understand are used by the code that provides the code snippets on our documentation site